### PR TITLE
Faster build kmer index

### PR DESCRIPTION
**Pull request by pjbradbury**
## Description

_Provide a summary of your changes including motivation and context.
If these changes fix a bug or resolves a feature request, be sure to link to that issue._

## Type of change

_What type of changes does your code introduce? Put an `x` in boxes that apply._

- [ ] `CHANGE` (fix or feature that would cause existing functionality to not work as expected)
- [ ] `FEATURE` (non-breaking change which adds functionality)
- [ ] `BUGFIX` (non-breaking change which fixes an issue)
- [x] `ENHANCEMENT` (non-breaking change which improves existing functionality)
- [ ] `NONE` (if none of the other choices apply. Example, tooling, build system, CI, docs, etc.)

## Checklist:

- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated relevant documentation

## Changelog entry

_Please add a one-line changelog entry below. This will be copied to the changelog file during the release process._

```release-note

```

### CODE REVIEW

**Comment by lynnjo** [comment]
Were any tests performed with larger datasets on cbsu to test timing changes between the original method and this branch, and to ensure grabbing larger chunks of data (each chrom) is not a memory issue?  

Do we need any document changes specifying memory needs for this step?

**Review by lynnjo — APPROVED** [review]

**Comment by pjbradbury** [comment]
A test is failing because of a missing file, which I just committed.

File: src/main/kotlin/net/maizegenetics/phgv2/pathing/FindPaths.kt [hunk 1]
```diff
@@ -84,7 +84,7 @@ class FindPaths: CliktCommand(help = "Impute best path(s) using read mappings.")
         option("--path-keyfile", help = "Name of tab-delimited key file.  Columns for samplename and filename" +
                 " are required. Files must be either read mapping files (ending in _readMapping.txt) or fastq files. " +
                 "If using paired end fastqs, a filename2 column can be included. A value must be entered for " +
-                "either --key-file or --read-files.")
+                "either --path-keyfile or --read-files.")
```
Comment (tcasstevens) [inline src/main/kotlin/net/maizegenetics/phgv2/pathing/FindPaths.kt:87]: There seems to be a few references to --key-file in imputation.md?

**Review by tcasstevens — APPROVED** [review]

File: src/main/kotlin/net/maizegenetics/phgv2/pathing/FindPaths.kt (hunk 1 above)
Comment (pjbradbury) [reply src/main/kotlin/net/maizegenetics/phgv2/pathing/FindPaths.kt:87]: --key-file is used by MapReads, while --path-key-file is used by FindPaths. I did find one place that needed to be changed

File: docs/imputation.md [hunk 2]
```diff
@@ -110,7 +120,7 @@ along with the cumulative coverage, which is the proportion of reads mapping to
 
 ### Required Parameters
 
-**--path-keyfile** or **--read-file** but not both. **--key-file** is the name and path of a keyfile that contains a list of the 
+**--path-keyfile** or **--read-file** but not both. **--path-keyfile** is the name and path of a keyfile that contains a list of the 
 read mapping files or read files (fastq). The keyfile must have two columns labeled sampleName and filename.
```
Comment (btmonier) [inline docs/imputation.md:124]: Capitalize all letters in "fastq" to "FASTQ"; Add backticks to "sampleName" and "filename"

File: docs/imputation.md [hunk 3]
```diff
@@ -110,7 +120,7 @@ along with the cumulative coverage, which is the proportion of reads mapping to
 
 ### Required Parameters
 
-**--path-keyfile** or **--read-file** but not both. **--key-file** is the name and path of a keyfile that contains a list of the 
+**--path-keyfile** or **--read-file** but not both. **--path-keyfile** is the name and path of a keyfile that contains a list of the 
 read mapping files or read files (fastq). The keyfile must have two columns labeled sampleName and filename. 
 Any additional columns will be ignored. SampleName must be unique.
 Additionally, if an output hvcf for a sample name already exists in the output directory, the sample will be skipped.
```
Comment (btmonier) [inline docs/imputation.md:126]: Change "hvcf" to "hVCF"

**Pull request merged**